src/examples/PairHMM/Makefile

```diff
@@ -0,0 +1,19 @@
+CC      = gcc
+CFLAGS= -O3 -std=c++11
+SHARED_LIBRARIES = -L../../../build/native -lgkl_pairhmm_c -lstdc++
```

Comment:
This Makefile is not sufficent to link the application more work is needed. 

It seem a header file is needed to use  gkl_pairhmm_c